app/components/in-view.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
```

Comment:
I can't reproduce these results. If you open the profiler and run a profile, you will notice the huge ui freeze when adding elements. This can also be observed in the chrome timeline. Here are some dumps: https://drive.google.com/folderview?id=0B_0L-A0ICAeeNlFockFzaWZVR28&usp=sharing